lib/src/api.dart

```diff
@@ -67,4 +74,244 @@ class AppStoreConnectApi {
     final response = await _client.get(request);
     return response.asList<Build>();
   }
+
+  Future<InAppPurchase> getInAppPurchase(String id) async {
+    final response = await _client.get(GetRequest(AppStoreConnectUri.v2(resource: 'inAppPurchases/$id')));
+    return response.as<InAppPurchase>();
+  }
+
+  Future<List<InAppPurchase>> getInAppPurchases({int limit = 200}) async {
+    final request = GetRequest(AppStoreConnectUri.v1(resource: 'apps/$_appId/inAppPurchasesV2'))
+      ..limit(limit);
+    print('uri: ${request.toUri()}');
+    final response = await _client.get(request);
+
+    return response.asList<InAppPurchase>()..sort((a, b) => a.productId.compareTo(b.productId));
+  }
+
+  Future<bool> postInAppPurchase(String productId, String name, String reviewNote, InAppPurchaseType type) async {
+    final body = {
+      'data': {
+        'type': 'inAppPurchases',
+        'attributes': {
+          'familySharable': false,
+          'inAppPurchaseType': type.toString(),
+          'name': name,
+          'productId': productId,
+          "reviewNote": reviewNote,
```

Comment:
[Anmerkung] Hier sollten wir überlegen, ob wir in einem öffentlichen Paket nicht alle Attribute vom Anwender konfigurierbar halten wollen. Das `familyShareable` sollten wir z.B. nicht immer auf `false` setzen. Diese könnten wir ähnlich zu den `AppStoreVersionAttributes` halten.

Wenn wir diesen Ansatz verfolgen könnten wahrscheinlich alle der jetzt fix implementierten Requests die `postModel` API des Clients nutzen und die Anpassung für andere Anwendungsfälle erlauben.

Reply:
done